### DISCUSSION TITLE
Individual fields can be boosted with the caret (^) notation at search time

### DISCUSSION THREAD

**Discussion by fromTechLead** [meilisearch/product#270] [Feedback & Feature Proposal]
Apply ranking rules at search time:  "Individual fields can be boosted with the caret (^) notation"

ElasticSearch analog: https://www.elastic.co/guide/en/elasticsearch//reference/current/query-dsl-multi-match-query.html

For example:
'SKU^3',
'manufacturer^2',
'name'

**Comment by gmourier** [comment]
Hi @DenisP01 👋,

Thanks for this feedback.

The main thing is to be able to send what we have today on the Settings side with the SearchableAttributes that define an order of importance but at the time of the search. Am I wrong?

**Comment by sandstrom** [comment]
For reference, here are a few more posts on basically the same thing:

- https://github.com/meilisearch/product/discussions/205
- https://github.com/meilisearch/product/discussions/216
- https://github.com/meilisearch/product/discussions/272

All of them are query time adjustments of how hits are scored, using e.g. functions (decay, or boosting individual fields, as mentioned here).

I'd probably go straight for something flexible, that caters to a lot of use-cases, if I were you, instead of doing something narrow like caret syntax.

**Reply by gmourier** [reply]
Thanks @sandstrom 💪

**Comment by macraig** [comment]
Hello everyone 👋

We just released a 🧪 prototype that allows **boosting results that match a custom filter**, and we'd love your feedback.

### How to use the prototype?
See https://www.notion.so/meilisearch/boostingFilter-API-usage-20aae06bc85e41dba828a90331a69f2c

⚠️ We do NOT recommend using this prototype in production. This is for test purposes only.

Feedback and bug reporting when using this prototype are encouraged! Thanks in advance for your involvement. It means a lot to us ❤️
